core/controllers/question_editor.py

```diff
@@ -64,14 +64,23 @@ def post(self, comma_separated_skill_ids):
             question = question_domain.Question.from_dict(question_dict)
         except Exception:
             raise self.InvalidInputException
+        skill_difficulties = self.payload.get('skill_difficulties')
+
+        if len(skill_ids) != len(skill_difficulties):
+            raise self.InvalidInputException
+
+        try:
+            skill_difficulties = [
+                float(difficulty) for difficulty in skill_difficulties]
+        except Exception:
```

Comment:
Perhaps: `except (ValueError, TypeError)`

(I assume you are catching conversion issues?)

Reply:
Done